modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_localized.js

```diff
@@ -703,7 +703,7 @@ AUI.add(
 				},
 
 				INPUT_HIDDEN_TEMPLATE:
-					'<input data-field-name={name} data-language-id={languageId} id="{namespace}{id}_{languageId}" name="{namespace}{fieldNamePrefix}{name}_{languageId}{fieldNameSuffix}" type="hidden" value="" />',
+					'<input data-field-name={name} data-languageid={languageId} id="{namespace}{id}_{languageId}" name="{namespace}{fieldNamePrefix}{name}_{languageId}{fieldNameSuffix}" type="hidden" value="" />',
```

Comment:
This is slightly scary, as there may be customers with code relying on `data-language-id`. This is a very old component. To be 100% extra safe and avoid dealing with breaking changes, can we set both `data-language-id={languageId}` and `data-languageid={languageId}`?

Reply:
This attribute was added here https://github.com/liferay-frontend/liferay-portal/pull/3913/commits/1d77abb6b3debec8fd3e2dd8bac3833a1cb88f03, so I think we don't have to worry about whether customers are using it, it's something very new

Reply:
Yeah, makes sense, the risk here seems extremely low :+1: